app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -111,6 +124,17 @@ def process_server_entity(server, entity)
         end
       end
 
+      def process_availability(availability)
+        case
+        when availability.nil?, availability['value'] == 'unknown'
```

Comment:
is it worth to check here perhaps with downcase? in case the value arrives as Uknown or UKNOWN